db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1880,7 +1880,7 @@ Status DBImpl::ReFitLevel(ColumnFamilyData* cfd, int level, int target_level) {
     Status status = versions_->LogAndApply(cfd, mutable_cf_options,
                                            read_options, write_options, &edit,
                                            &mutex_, directories_.GetDbDir());
-
+    c->MarkFilesBeingCompacted(false);
```

Comment:
Seem like an opportunity for future tech debt to bundle related book-keeping together (e.g, release,unregister,MarkFilesBeingCompacted(false), destructor stuff) ...

Reply:
I tried to use `Compaction::ReleaseCompactionFiles()` here which is more consistent with other places. It failed since in ReFitLevel() we don't call Compaction::FinalizeInputInfo() which sets cfd_ in the compaction, while ReleaseCompactionFiles uses cfd_.